lib/tasks/application.rake

```diff
@@ -121,4 +121,43 @@ namespace :application do
     Rails.logger = ActiveSupport::Logger.new($stdout)
     Rails.logger.level = 1
   end
+
+  namespace :links_valid do
+    desc "Checks the validity of links in division summary"
+    task divisions: :environment do
+      base_url = "https://theyvoteforyou.org.au"
```

Comment:
Ideally we'd get this from a configuration rather than hardcoding it but it looks like we don't actually have a configuration that we can trivially use. Other OAF projects have something like this but not here unfortunately yet.

Reply:
Could you refer me to an OAF project with the appropriate configuration?

Reply:
Is this even necessary anymore? It is now removed after using URL helpers

Reply:
I'm not sure off the top of my head. With the url helpers in normal views it gets the domain from the web request. For mailers it gets the domain from the configuration [here](https://github.com/openaustralia/publicwhip/blob/master/config/environments/production.rb#L69). I'm not totally sure what happens in a rake task off the top of my head